src/viewport_renderer.rs

```diff
@@ -1,13 +1,13 @@
 use std::collections::HashMap;
 use std::convert::TryFrom;
 use std::error;
+use std::f32;
```

Comment:
The same as in one of the previous comments. Is this needed for anything?

Reply:
And I found one more occurence in the very first file, but I'm now lazy to rearrange the comments.

Reply:
Nope, not in this file. Weird that there is no unused lint. I'll remove.

Reply:
Done.